package.json

```diff
@@ -10,6 +10,7 @@
     "@rollup/plugin-commonjs": "^14.0.0",
     "@rollup/plugin-node-resolve": "^8.0.0",
     "rollup": "^2.3.4",
+    "rollup-plugin-css-only": "^3.0.0",
     "rollup-plugin-livereload": "^2.0.0",
     "rollup-plugin-svelte": "^6.0.0",
```

Comment:
```suggestion
    "rollup-plugin-svelte": "^7.0.0",
```

Reply:
done. I also removed `emitCss` from `rollup.config.js` so that we can rely on the default